.clang-format

```diff
@@ -1,27 +1,30 @@
 ---
-Language: Cpp
-# BasedOnStyle: LLVM
+Language:        Cpp
```

Comment:
I can't remember whether I had copied in the .clang-format from gnuradio or gotten it somewhere else.  Is there specific motivation to the changes?

Reply:
The file did not seem to work with Ubuntu 20.04's default clang version and the provided .clang-format until I did this. We could just leave it as it is and allow developers to have modified versions locally.